src/editor.tsx

```diff
@@ -8,7 +8,7 @@ import "codemirror/lib/codemirror.css";
 import { WebsocketProvider } from "y-websocket";
 
 const ROOM_NAME = "new-room";
-const SIGNALLING_SERVICE = process.env.STORYBOOK_SIGNAL_URL|| 'ws://localhost:6001';
+const SIGNALLING_SERVICE = process.env.STORYBOOK_SIGNAL_URL|| 'ws://localhost:6006/ws';
```

Comment:
Why did the port number change to 6006?

Reply:
Because the JS bundle and and the websocket service are served from the same url. storybook is working as a proxy.

See ecfc1742ae2d6629ecce3e62ff2c9b1b96523da1 for details